hangman.py

```diff
@@ -0,0 +1,300 @@
+    return "".join(compared_list)
+
+
+def get_available_letters(letters_guessed):
+    """
+    letters_guessed: set (of letters), which letters have been guessed so far
+    returns: string (of letters), comprised of letters that represents which letters have not
+      yet been guessed.
+    """
+    alphabets = set(string.ascii_lowercase)
+    compared_list = [i for i in alphabets if i not in letters_guessed]
+    return "".join(compared_list)
+
+
+def count(warn, tries):
+    """
+    warn: integer, amount of warnings
+    tries: integer, amount of tries
+    This function deletes one warning or a try if there isn't any tries.
+    returns: tuple with amount of warnings and tries.
+    """
+    if warn > 0:
+        warn -= 1
+    else:
+        warn -= 1
+        tries -= 1
+    return warn, tries
+
+
+def vowel_check(letter, tries):
+    """
+        letter: string, entered letter
+        warn: integer, amount of warnings
+        tries: integer, amount of tries
+        This function deletes two tries if entered letter is a vowel, or one if not.
+        returns: amount of tries.
+    """
+    vowels = {"a", "e", "i", "o", "u"}
+    if letter in vowels:
+        tries -= 2
+    else:
+        tries -= 1
+    return tries
+
+
+def check_element(tries, warn, letters_guessed, word, letter, hints):
+    """
+        tries: integer, amount of tries
+        warn: integer, amount of warnings
+        letters_guessed: list with elements which have been entered so far
+        word: string, comprised of letters, underscores (_), and spaces that represents which letters in secret_word
+        have been guessed so far
+        letter: string, entered symbol
+        hints: boolean, which shows if hints are turned on or off
+        This function checks if the entered symbol is a letter. If it's a letter, the program shows if the letter is
+        guessed. If entered symbol was entered before or it isn't a letter, program shows a warning.
+        returns: tuple with amount of tries and warnings, list with elements which have been entered so far and string,
+                 comprised of letters, underscores (_), and spaces that represents which letters in secret_word have
+                 been guessed so far
+    """
+    # If the entered symbol has already been used, this part shows a warning
+    if letter in letters_guessed:
+        warn, tries = count(warn, tries)
+        # This output changes depending on if the entered symbol is "*" or a letter and how many warnings are left
+        print("Oops! You've already", *["used your hint." if letter == "*" else "guessed that letter."], "You have",
+              *[warn if warn >= 0 else "no"],
+              "warnings left" + str(*[" so you lose one guess" if warn < 0 else ""]) + ":", word)
+
+    # If the entered symbol is a "*", the program uses a hint and shows all possible matches
+    elif hints and letter == "*":
+        show_possible_matches(word)
```

Comment:
On my opinion, you shouldn't perform it in "check_element" function, as it's not a symbol that you need to check whether it's a valid symbol or not. I'd perform that on 1 level up (just in the main loop)